src/AxiomTest.sol

```diff
@@ -85,11 +86,47 @@ abstract contract AxiomTest is Test {
         axiomVm = new AxiomVm(axiomV2QueryAddress, urlOrAlias);
     }
 
+    /// @dev Create a forked test environment from the latest block and set up crosschain Axiom contracts
+    /// @param urlOrAlias The URL or alias of the fork to create
+    /// @param sourceChainId The chain ID of the source chain
+    /// @param isBlockhashOracle Whether the bridging is via a blockhash oracle
+    /// @param bridgeId The ID of the bridge
+    function _createSelectForkAndSetupAxiomCrosschain(
+        string memory urlOrAlias,
+        uint64 sourceChainId,
+        bool isBlockhashOracle,
+        uint8 bridgeId
```

Comment:
feels kind of ugly to have this here if it's not used if `isBlockhashOracle` is `true`, but I guess it does seem like the better choice between creating separate functions for broadcaster and blockhashOracle...

Reply:
Yeah, not great -- lack of optional in Solidity